HACKING.md

```diff
@@ -35,67 +35,72 @@ Enter the toolbox with
 
 Cockpit uses a lot of build and runtime dependencies from [npm](https://www.npmjs.com). Get them with
 
     tools/node-modules checkout
 
-Next run this command from your top level Cockpit checkout directory, and make
+Next, run this command from your top level Cockpit checkout directory, and make
 sure to run it as the same user that you'll use to log into Cockpit below.
 
     mkdir -p ~/.local/share/
     ln -s $(pwd)/dist ~/.local/share/cockpit
 
-This will cause cockpit to read JavaScript and HTML files directly from the
-built package output directory instead of using the installed Cockpit UI files.
+This will cause cockpit to read JavaScript, HTML, and CSS files directly from the
+locally built package output directory instead of using the system-installed Cockpit
+files.
 
 Now you can log into Cockpit on your local Linux machine at the following
 address. Use the same user and password that you used to log into your Linux
 desktop.
 
 <http://localhost:9090>
 
-After every change to your sources the bundles need to be rebuilt. The
-recommended and fastest way is to do that in "watch" mode on the particular
-page that you are working on, see [pkg/](./pkg/) for which pages exist.
-E.g. if you want to work on anything in [pkg/systemd](./pkg/systemd/), run:
+After every change to the source files, bundles need to be rebuilt. The
+recommended and fastest way is to do that is using the "watch" mode on 
+the page that you are working on. For example, if you want to work on 
+anything in [pkg/systemd](./pkg/systemd/), run:
 
     tools/webpack-watch systemd
 
-Note that this enables eslint by default -- if you want to disable it, run it
-with `-e`/`--no-eslint`.
+See [pkg/](https://github.com/cockpit-project/cockpit/tree/main/pkg)
+for a list of all pages.
 
-Then reload cockpit in your browser after page is built. Press Control-C to
+Note that this enables eslint by default -- if you want to disable JavaScript
+warnings, run the command with `-e`/`--no-eslint`.
+
+Reload cockpit in your browser after page is built. Press `Ctrl`-`C` to
 stop watch mode once you are done with changing the code.
 
-Often you need to test your code in a VM. There is an `-r`/`--rsync`
+You often need to test code changes in a VM. There is an `-r`/`--rsync`
 option for copying the built page into the given SSH target's
-/usr/share/cockpit/ directory. If you use cockpit's own test VMs and set up the
+/usr/share/cockpit/ directory. If you use Cockpit's own test VMs and set up the
 SSH `c` alias as described in [test/README.md](./test/README.md), you can use
 one of these commands:
 
     tools/webpack-make.js kdump -r c
     tools/webpack-watch kdump -r c
 
-To make Cockpit again use the installed code, rather than that from your git
-checkout directory, run the following, and log into Cockpit again:
+If you'd like Cockpit to use the system-installed package instead of your
+development checkout, run the following, and then log into Cockpit again:
```

Comment:
This lost this "revert previous change" flavor that the previous "again" had. If you don't like that, perhaps "If you like to go back to using the system-installed pages.."?

Reply:
I've made a third version. How's this?

https://github.com/cockpit-project/cockpit/pull/18509/files#diff-9769a3f5504af3cd735e08f078b94f827bdd51eacac2e9d80ac9761bc485171fL90-R96

> To make Cockpit use system packages again, instead of your checkout directory, remove the symlink with the following command and log back into Cockpit: